data/presets/playground/trampoline.json

```diff
@@ -0,0 +1,11 @@
+{
+    "icon": "maki-playground",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "playground": "trampoline"
+    },
+    "name": "Play Trampoline"
```

Comment:
Is there any other kind? 🙃

Reply:
I was trying to distinguish this from backyard trampolines, but after looking back at the wiki article, I noticed it doesn't say it has to be on a playground.